labs/3.md

```diff
@@ -32,6 +32,7 @@ EOF
 The KubeFed Controller should be deployed now:
 
 ~~~sh
+wait-for-deployment cluster1 kube-federation-system kubefed-controller-manager
```

Comment:
is this in the wrong spot?

Reply:
No, it's okay there. We should wait until the deployment is ready in order to see the pods running.